packages/launchpad/src/use-tracking/index.tsx

```diff
@@ -0,0 +1,71 @@
+import { recordTracksEvent } from '@automattic/calypso-analytics';
+import { SiteDetailsOptions } from '@automattic/data-stores';
```

Comment:
Nit: this should be a type import.
```suggestion
import type { SiteDetailsOptions } from '@automattic/data-stores';
```

Reply:
Fixed here https://github.com/Automattic/wp-calypso/pull/87141/commits/a5b2c5fe311fb03a6ef09a4acf3caa197320b3f4

(I can't just accept the suggestion because the linter likes to control the import order 😃 )